CHANGELOG.md

```diff
@@ -9,7 +9,19 @@ _With the release of `v3.0.0`, we're introducing a new changelog format in an at
 
 _The old changelog can be found in the `release-2.6` branch_
 
-# Changes Since v3.4.0
+# Changes Since v3.4.1
+
+# v3.4.1 - [2019.09.17]
+
```

Comment:
I'd like to see links to PRs or Issues here, but I don't think it has to be done.

Reply:
I was thinking that too at first... as that was done in old 2.x changelogs I last was involved in.

However, in CHANGELOG.md we don't seem to do that now? I was following the precedent of the last point release... `3.1.1`, and there don't seem to be any links at all in the changelog these days